src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -23,7 +23,9 @@
  */
 public class ParserUtil {
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INDEX_TOO_LONG = "Index provided must be no longer than 4 characters";
```

Comment:
perhaps you can explain here that NUScheduler only supports up to 9999 entries. good to mention in UG as well

Reply:
For this error message, it does not get displayed to users as it will be caught in the respective command parsers and the error message shown will be invalid command format.

Maybe for this I will add into the section `Notes about the command format` to explain that only up to 9999 entries is supported which sufficiently meets the needs of most NUS students?